packages/npm/send/example/src/App.tsx

```diff
@@ -15,52 +23,110 @@ export const serializeError = (error: unknown) => {
   }
 }
 
-export default function App() {
-  const [result, setResult] = React.useState<string>('waiting...')
+const App = () => {
+  const [data, setData] = useState<{
+    trackName: string
+    artistName: string
+    artworkUrl100: string
+    description: string
+    version: string
+    screenshotUrls: string[]
+  } | null>()
 
   React.useEffect(() => {
     send({
       baseURL: 'https://itunes.apple.com',
       method: 'GET',
       path: '/lookup',
-      queryParameters: {
-        bundleId: 'com.trycandle.candle',
-        country: 'US',
+      query: {
+        parameters: {
+          bundleId: 'com.trycandle.candle',
+          country: 'US',
+        },
       },
       body: null,
       utf8ContentTypes: ['application/json', 'text/html', 'text/javascript'],
-      headerParameters: {
-        'Cache-Control': 'no-cache',
-        'Content-Type': 'application/json',
-        'Accept': 'application/json',
+      header: {
+        parameters: {
+          'Cache-Control': 'no-cache',
+          'Content-Type': 'application/json',
+          'Accept': 'application/json',
+        },
       },
+    }).then((result) => {
+      if (result.error) {
+        console.log('ERROR', serializeError(result.error))
+      } else {
+        setData(JSON.parse(result.response?.body as any).results[0])
+      }
     })
-      .then((response) => {
-        console.log('SUCCESS', response)
-        setResult(response.body ?? 'no body')
-      })
-      .catch((error) => {
-        console.log('FAILURE', serializeError(error))
-        setResult(`[${error.code}] ${error.message}`)
-      })
   }, [])
 
+  if (!data) {
```

Comment:
```suggestion
  if (data === null) {
```
Falsiness/truthiness checks in TS are overly broad, have unintuitive behavior, and should not be used